jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java

```diff
@@ -167,7 +167,7 @@ public void testServletContext(String mode) throws Exception
     @Test
     public void testWebappContext() throws Exception
     {
-        Server server = new Server(8080);
+        Server server = new Server(0);
```

Comment:
The PR is about infinispan, but this is a change to a Weld cdi test?

Reply:
I have updated the PR title to explain the goal of this PR. 
Please note each change (testing if docker available and one for the hardcoded port) are already different commits.
So having 2 PRs will have the same result at the end with 2 different commits in the git history and each will have a correct description of the change.